build.gradle

```diff
@@ -269,7 +269,7 @@ dependencies {
   integrationTestCompile sourceSets.test.runtimeClasspath
   integrationTestCompile group: 'org.apache.sshd', name: 'sshd-scp', version: '2.4.0'
   integrationTestCompile group: 'org.apache.sshd', name: 'sshd-sftp', version: '2.4.0'
-  integrationTestCompile group: 'org.testcontainers', name: 'postgresql', version: '1.14.1' // brings in junit4. testcontainers depend on it
+  integrationTestCompile group: 'org.testcontainers', name: 'postgresql', version: '1.14.2' // brings in junit4. testcontainers depend on it
```

Comment:
can actually be "fixed" by excluding junit4 and including jupiter extension from same group. not sure whether even need to exclude junit4 right now. everything migrates to v5 atm